src/components/pages/mypage/UserProfileCard.tsx

```diff
@@ -26,60 +28,73 @@ export default function UserProfileCard({
   leaguePoints,
   profileImage,
 }: UserCardProps) {
+  const { data: myInfo } = useAuth();
+  const disclosure = useDisclosure();
+  const handleChangePasswordOpen = () => {
+    if (myInfo?.data.email) {
+      disclosure.onOpen();
+    }
+    alert('form 가입 사용자가 아닙니다.');
+    return;
+  };
```

Comment:
```suggestion
  const handleChangePasswordOpen = () => {
    // 이메일 가입 유저인 경우에만(`myInfo.data.email`이 nullable이 아닌 경우에만) 비밀번호 변경 가능
    if (myInfo?.data.email) {
      disclosure.onOpen();
    } else {
      alert('form 가입 사용자가 아닙니다.');
    }
  };
```
- 현재 코드는 이메일 가입 사용자가 아닌 경우에도 alert를 띄우는 거 같습니다.
- 위와 같이 관련 주석을 추가하면 좋을 것 같습니다.

Reply:
아 끝맺음 처리를 제대로 못했네요.